src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -205,9 +216,21 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
           </Button>
         </div>
       )} */}
+
+      <Toggler
+        name="bookmarked-toggle"
+        toggled={showPersonalisedSchedule}
+        onChange={() => setShowPersonalisedSchedule()}
```

Comment:
will it work this way? and I think I'd rename it to `togglePersonalisedSchedule`
```suggestion
        onChange={setShowPersonalisedSchedule}
```

Reply:
fixed in 2ff22f0e2bc731fcf9bbafbeeefe18dce293c065